helm/draughtsman-operator-chart/templates/secret.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: draughtsman-operator-secret
+  namespace: draughtsman
+data:
+  secret.yaml: {{ .Values.Installation.V1.Secret.DraughtsmanEventer.SecretYaml | b64enc | quote }}
```

Comment:
I am just using the eventer stuff now because the operator stuff is not set up. Will fix this later as soon as there are actual secrets for the operator.